src/NodeRTLib/CppTemplates/Event.cpp

```diff
@@ -70,16 +70,16 @@
 
       if (info.Length() < 2 || !info[0]->IsString() || !info[1]->IsFunction())
       {
-        Nan::ThrowError(Nan::Error(NodeRT::Utils::NewString(L"wrong arguments, expected a string and a callback")));
+        Nan::ThrowError(Nan::Error(NodeRT::Utils::NewString("wrong arguments, expected a string and a callback")));
         return;
       }
 
       String::Value eventName(info[0]);
```

Comment:
(Applies to other uses of CaseInsensitiveEquals as well - we should always make sure that we compare between the 2 types of strings)